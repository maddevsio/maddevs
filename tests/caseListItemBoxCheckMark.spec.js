import { mount } from '@vue/test-utils'
import ListItemBoxCheckMark from '@/components/Cases/ListItemBoxCheckMark'

describe('list item box checkMark', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(ListItemBoxCheckMark, {
      slots: {
        default: 'Main Content',
      },
    })
  })

  // ------ IMPORTANT ----- //
  it('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
  // --------------------- //

  it('should have correct data in slot', () => {
    expect(wrapper.html()).toContain('Main Content')
  })
})
