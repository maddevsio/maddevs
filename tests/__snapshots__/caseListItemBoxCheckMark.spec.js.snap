// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list item box checkMark renders correctly 1`] = `
<li
  class="list-item_box"
>
  Main Content
</li>
`;
